### DISCUSSION TITLE
Option ->merge('field-to-match')

### DISCUSSION THREAD

**Discussion by FrodeMMeling** [konnco/filament-import#69] [Ideas]
If the importfile contains a record that already exist and a column has the unique constraint. it crashes.. It would have been nice to have an option like the required(), where you state mergeEqualsBasedOn('field').

Then if the column in excel matches a record you merge changes instead of failing.

Then also a "ignoreEqualsBasedOn" can maybe also be a feature?

**Comment by frankyso** [comment]
That's great idea, ill try to cover it when im available, or you can free to create a PR for the feature 😊
